_observing-your-data/trace/ta-dashboards.md

```diff
@@ -1,25 +1,143 @@
+
+---
+
+## Schema dependencies and assumptions
+
+The plugin requires [Data Prepper]({{site.url}}{{site.baseurl}}/data-prepper/) to process and visualize OTel data and relies on the following Data Prepper pipelines for OTel correlations and service map calculations:
+
+- [Trace analytics pipeline]({{site.url}}{{site.baseurl}}/data-prepper/common-use-cases/trace-analytics/)
+- [Service map pipeline]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/processors/service-map-stateful/)
+
+### Standardized telemetry data
+
+The plugin requires telemetry data to follow the OTel schema conventions, including the structure and naming of spans, traces, and metrics as specified by OTel, and to be implemented using [Simple Schema for Observability (SS4O)]({{site.url}}{{site.baseurl}}/observing-your-data/ss4o/).
+
+### Service names and dependency map
+
+For accurate service mapping and correlation analysis, adhere to the following guidelines:
+
+- Service names must be unique and used consistently across application components.
+- The `serviceName` field is populated using the Data Prepper pipeline.
+- Services must be ingested with predefined upstream and downstream dependencies to construct accurate service maps and understand service relationships.
+
+### Trace and span IDs
+
+Traces and spans must have consistently generated and maintained unique identifiers across distributed systems to enable end-to-end tracing and accurate performance insights.
+
+### RED metrics adherence
+
+The plugin expects metrics data to include rate, error, and duration (RED) indicators for each service, either pre-aggregated using the Data Prepper pipeline or calculated dynamically based on spans. This allows for effective computation and display of key performance indicators.
+
+### Correlation fields
+
+Certain fields, such as `serviceName`, must be present for correlation analysis. These fields enable the plugin to link related telemetry data and provide a holistic view of service interactions and dependencies.
+
+### Correlation indexes
+
+Navigating from the service dialog to its corresponding traces or logs requires the existence of correlating fields and that the target indexes (for example, logs) follow the naming conventions, as described in [Simple Schema for Observability](https://opensearch.org/docs/latest/observing-your-data/ss4o/).  
+
+---
+
+## Trace analytics with OTel protocol analytics
+Introduced 2.15
+{: .label .label-purple }
+
+Trace analytics with OTel protocol analytics in OpenSearch Dashboards provides comprehensive insights into distributed systems. OpenSearch Dashboards gives you a capabilities to correlate signals, identify relationships, and uncover root causes of issues in a single interface. You can analyze and visualize the following:
```

Comment:
Nit: Can we include the noun after "the following"?

Reply:
"assets"